sdk/test/test_kfserving_v1alpha1_custom_spec.py

```diff
@@ -0,0 +1,40 @@
+# coding: utf-8
+
+"""
+    KFServing
+
+    Python SDK for KFServing  # noqa: E501
+
+    OpenAPI spec version: v0.1
+    
+    Generated by: https://github.com/swagger-api/swagger-codegen.git
+"""
+
+
+from __future__ import absolute_import
+
+import unittest
+
+import kfserving
+from kfserving.models.kfserving_v1alpha1_custom_spec import KfservingV1alpha1CustomSpec  # noqa: E501
+from kfserving.rest import ApiException
+
+
+class TestKfservingV1alpha1CustomSpec(unittest.TestCase):
+    """KfservingV1alpha1CustomSpec unit test stubs"""
+
+    def setUp(self):
+        pass
+
+    def tearDown(self):
+        pass
+
+    def testKfservingV1alpha1CustomSpec(self):
```

Comment:
not sure what these tests are supposed to do since we are just going to call `CustomObjectApi` to create KFService.

Reply:
I suggest keeping that since that's generated by swagger-codegen, at least the case can test the import/load has no problem. Anyway, we need to make and clear up the test cases after we developed some action apis :-）